### ISSUE TITLE
Create dataset loader for ViHealthQA

### ISSUE THREAD

**Issue by SamuelCahyawijaya** [SEACrowd/seacrowd-datahub#306]
Dataloader name: `vihealthqa/vihealthqa.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?vihealthqa

|     Dataset| vihealthqa |
|-------------|---|
| Description | The ViHealthQA dataset consists of 10,015 question-answer passage pairs. The questions originate from health-interested users and were asked on prestigious health websites, while the answers come from highly qualified experts. The dataset is divided into three parts: the training set, validation set, and test set, each containing 7,009, 993, and 2,013 question-answer pairs, respectively. |
| Subsets | - |
| Languages | vie |
| Tasks | Question Answering |
| License | Unknown (unknown) |
| Homepage | https://huggingface.co/datasets/tarudesu/ViHealthQA |
| HF URL | https://huggingface.co/datasets/tarudesu/ViHealthQA |
| Paper URL | https://link.springer.com/chapter/10.1007/978-3-031-10986-7_30 |

**Comment by Gyyz** [comment]
#self-assign
